### ISSUE TITLE
Get rid of the timer during the nothing phase

### ISSUE THREAD

**Issue by millette** [remvze/nothing#5]
From <https://news.ycombinator.com/item?id=41555806>:

> Love it!
> One minor improvement idea: get rid of the timer during the nothing phase. I looked at it all the time.
> You could show the timer once someone starts scrolling again.

**Comment by remvze** [comment]
I don't think that would work with the way Nothing functions right now, since scrolling is supposed to reset the timer. Showing it only then would always display zero seconds.

**Comment by millette** [comment]
It would still update the record. I understand the comment on HN, the counter is a little bit distracting.
